README.md

```diff
@@ -196,6 +197,10 @@ your IBM Cloud Cluster API Kubernetes cluster.
    rm -rf $HOME/.ssh/id_ibmcloud*
    ```
 
+### Create additional worker nodes
+
+Please refer to [Trouble shooting documentation](docs/add_node.md) for further info.
```

Comment:
Can you put the document in this file directly? I think adding a node should not belong to troubleshooting section.

Reply:
a typo..